v2/support/garden_sample_app/assets/iframe.html

```diff
@@ -2,7 +2,7 @@
 <head>
   <!-- Zendesk Garden styles are provided for apps from this URL. We recommend hotlinking to this version
        provided by our CDN to ensure the latest updates are refected in your app. -->
-  <link rel="stylesheet" href="https://assets.zendesk.com/apps/sdk-assets/css/1/zendesk_garden.css" type="text/css">
+  <link rel="stylesheet" href="https://assets.zendesk.com/apps/sdk-assets/css/latest/zendesk_garden.css" type="text/css">
```

Comment:
Even here, I'm a bit concerned. It's telling app developers this is available and can be used. If we do this we'll need to change the comment above to explicitly say we do NOT recommend pointing to `latest` as this version will be updated with breaking changes without notice. cc @dpawluk @Eshen

Reply:
Fair enough, addressed. Does this work?

Reply:
I would recommend pointing to a specific major version, so they get the patch and minor updates automatically (e.g. /2/zendesk_garden.css). Other than that it seems ok. Though what do you think is the advantage of keeping latest? Why not refer to the recommended url instead? We'll need to keep updating these docs in any case...

Reply:
I included `latest` here as it seemed sensible to me to demo the latest updates to the Garden styles in the demo app. Updated to point to a major version though 👍

Reply:
@fvzzy if we point it to 2 it will be the latest version in any case, and it will be clear to the developer which url they should use.

Reply:
This actually seems like a legitimate use case for `latest` to me - it seems silly to publish to it at all just to bury it completely. I've updated it to 2 and removed the comment in any case

Reply:
...except this time with the actual work committed 💇